containers/app/app.py

```diff
@@ -0,0 +1,56 @@
+import os
+import subprocess
+import connexion
+import sqlalchemy
+from shapely.geometry import Polygon
+from shapely import wkt
+
+
+DB_PASSWORD = os.environ['POSTGRES_PASSWORD']
+RAW_DIR = '/osm/pbf'
+CONVERTED_DIR = '/osm/pbf/converted'
+MAPCONFIG_PATH = '/osm/mapconfig.xml'
+engine = sqlalchemy.create_engine(f'postgresql+psycopg2://postgres:{DB_PASSWORD}@pgrouting/postgres')
+
+
+class Loader:
+    @staticmethod
+    def create_dirs(directory):
+        if not os.path.exists(directory):
+            os.makedirs(directory)
```

Comment:
There is an inconsisency between function name and what it does. Accordin to it's name this function should create multiple directories, but it creates only one directory. I suggest to rename this function/
```suggestion
    @staticmethod
    def create_directory(path_to_directory: str):
        if not os.path.exists(path_to_directory):
            os.makedirs(path_to_directory)
```

Reply:
os.makedirs() method in Python is used to create a directory recursively. Actually, it looks good, but I agree that name should be more clear in terms of taken goal